### PR TITLE
MAUI-5125: Add note for Corner Radius property.

### PR DESCRIPTION
**Pull request by RasikaPalanisamy**
Added note for CornerRadius property.

### CODE REVIEW

File: MAUI/Popup/styles.md [hunk 1]
```diff
@@ -247,6 +247,8 @@ The SfPopup allows customizing the stroke appearance using the following propert
 </tr>
 </table>
 
+N>On Android 33 and above, it is possible to set different corner radii for each corner using the [CornerRadius](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.PopupStyle.html#Syncfusion_Maui_Popup_PopupStyle_CornerRadius) class. However, on versions below Android 33, if the same value is provided for all corners, a corner radius will be applied. If different values are provided for each corner, the corner radius may not be applied.
```
Comment (Chrispineagunja) [inline MAUI/Popup/styles.md:250]: N>On Android 33 and above, it is possible to set different corner radii for each corner using the [CornerRadius](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.PopupStyle.html#Syncfusion_Maui_Popup_PopupStyle_CornerRadius) class. However, on versions below Android 33, a corner radius will be applied if the same value is provided for all corners. The corner radius may not be applied if different values are provided for each corner.

Comment (RasikaPalanisamy) [reply MAUI/Popup/styles.md]: Addressed the review concerns provided.

**Review by Lakshminatarajan — APPROVED** [review]
Approved

**Review by jayaleshwari — APPROVED** [review]
This is fine

**Pull request merged**